components/automate-chef-io/content/docs/configuration.md

```diff
@@ -698,6 +710,37 @@ Then run `chef-automate config patch </path/to/your-file.toml>` to deploy your c
 # name = "automate"
 ```
 
+#### Configuring An External PostgreSQL Database
+To use Chef Automate with an external PostgreSQL cluster, create a `.toml` file containing the following partial configuration.
```

Comment:
🤔I'm wondering if we prefer "TOML file" to  "`.toml` file"?

Reply:
We're not consistent, so I'm going to make us consistent by making it "TOML file".